### PR TITLE
Update ReadMe.md

### LINKED ISSUE #5636: Remove duplication from docs
### Kind of issue 
  - [ ] Improvement
  - [ ] Bug
  - [x] Other, namely: Documentation

### This issue concerns 
  - [x] The NPM Package
  - [ ] The Packagist Package

### Description

Currently, the [Node Usage](https://github.com/simple-icons/simple-icons#node-usage) section of the documentation contains two nearly identical examples. This can 1) be confusing and 2) lead to inconsistencies when it needs to be updated. My suggestion would be to somehow merge them so the large comment (with the example icon object) is presented only once.

### PR DESCRIPTION
**Pull request by garyilayev**
**Issue:** Resolves #5636

Removed a duplicate example inside of Node Usage.

### Description
  - Removed a duplicate example, Only show icon object once.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,7 +91,7 @@ console.log(icon);
 }
 
 NOTE: the `guidelines` entry will be `undefined` if we do not yet have guidelines data for the icon.
```
Comment (PeterShaggyNoble) [inline README.md:93]:
```suggestion
NOTE: the `guidelines` entry will be `undefined` if we do not yet have guidelines for the icon.
```

File: README.md [hunk 2]
```diff
@@ -87,6 +67,11 @@ require('simple-icons/icons/[ICON SLUG]');
 
 // For example:
 const icon = require('simple-icons/icons/simpleicons');
+```
+
+Either method will return an icon:
```
Comment (PeterShaggyNoble) [inline README.md:72]:
```suggestion
Either method will return an icon object:
```

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
Welcome to Simple Icons, @garyilayev and thanks for taking this on 🙂 Couple of minor suggested edits below but otherwise this looks good to me.

**Comment by garyilayev** [comment]
Glad I can help :+1: : .

**Review by PeterShaggyNoble — APPROVED** [review]
Thanks, @garyilayev, that all looks good to me now 👍🏻

**Pull request merged**